### PR TITLE
feat: added fullscreen button

### PR DESCRIPTION
**Pull request by NathanDimmer**
Add a fullscreen button to the Maze activity.

Fixes issue #400 

@FreddieN @eohomegrownapps @utkarsh-raj @llaske @ashish0910

### CODE REVIEW

File: activities/MazeWeb.activity/js/activity.js [hunk 1]
```diff
@@ -53,6 +53,24 @@ define(["sugar-web/activity/activity","tween","rAF","activity/directions","sugar
...
+		document.getElementById("unfullscreen-button").addEventListener('click', function() {
+			document.getElementById("main-toolbar").style.opacity = 1;
+			document.getElementById("canvas").style.top = "55px";
+			document.getElementById("unfullscreen-button").style.visibility = "hidden";
+			onWindowResize();
+		});
+
 		var generateXOLogoWithColor = function(color) {
 			var coloredLogo = xoLogo;
 			coloredLogo = coloredLogo.replace("#010101", color.stroke)
```
Comment (FreddieN) [inline activities/MazeWeb.activity/js/activity.js]: Indentation here is out of line.

Comment (FreddieN) [reply activities/MazeWeb.activity/js/activity.js]: The generateXOLogoWithColor needs to move back as well.

File: activities/MazeWeb.activity/js/activity.js [hunk 2]
```diff
@@ -53,6 +53,24 @@ define(["sugar-web/activity/activity","tween","rAF","activity/directions","sugar
             }
         });
 
+        document.getElementById("fullscreen-button").addEventListener('click', function() {
+            document.getElementById("main-toolbar").style.opacity = 0;
+            console.log("1");
```
Comment (llaske) [inline activities/MazeWeb.activity/js/activity.js]: Don't forget to remove debugging messages

File: activities/MazeWeb.activity/js/activity.js (hunk 1 above)
Comment (NathanDimmer) [reply activities/MazeWeb.activity/js/activity.js]: All set! Thank you.

File: activities/MazeWeb.activity/js/activity.js (hunk 2 above)
Comment (NathanDimmer) [reply activities/MazeWeb.activity/js/activity.js]: Sorry! Fixed now.

**Comment by llaske** [comment]
Nice work. Thanks.

**Pull request merged**